src/components/Calls/styles.scss

```diff
@@ -73,6 +68,71 @@
 	height: 50%;
 
 	&__content {
-		height: 100%;
+		width: 100%;
+		height: 200%;
+	}
+}
+
+.joinCall {
+	width: 300px;
+	margin: 15px;
+
+	padding: 5px;
+
+	border: 1px solid #e4e7ea;
+
+	&__content {
+		display: flex;
+		flex-direction: row;
+
+		padding: 15px;
+
+		font-size: 15px;
```

Comment:
Again, I'm not sure why are keep changing font properties. We need to keep font properties consistent throughout the application